roles/initialisation/tasks/main.yml

```diff
@@ -62,10 +62,10 @@
     force: yes
     backup: yes
 
-- name: Replace OpenStack cloud config template with one supporting new AZ feature
-  become: true
-  copy:
-    src: templates/openstack.conf.j2
-    dest: /usr/share/ansible/openshift-ansible/roles/openshift_cloud_provider/templates/openstack.conf.j2
-    force: yes
-    backup: yes
+#- name: Replace OpenStack cloud config template with one supporting new AZ feature
```

Comment:
We should probably just remove this task altogether since it's no longer needed.

Reply:
Agreed. Done.